### PR TITLE
Adding books routes and endpoints

### PR DESCRIPTION
**Pull request by gabybrow**
## Summary
- Adding books routes and endpoints

## Trello Card
https://trello.com/c/vDDOOhhB/24-crear-rutas-index-y-show-para-book

### CODE REVIEW

File: spec/controllers/api/v1/books_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
```
Comment (Dinuuu) [inline spec/controllers/api/v1/books_controller_spec.rb]: Delete this file if you are not going to use it

File: app/controllers/api/v1/books_controller.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1
+    class BooksController < ApiController
+      def index
+        @books = Book.all
+        render json: @books
```
Comment (Dinuuu) [inline app/controllers/api/v1/books_controller.rb:6]: are you sure this is returning the JSON that is specified in Apiary ?

File: app/controllers/api/v1/books_controller.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      end
+
+      def show
```
Comment (Dinuuu) [inline app/controllers/api/v1/books_controller.rb:9]: idem Apiary

**Review by Dinuuu — CHANGES_REQUESTED** [review]

File: app/controllers/api/v1/books_controller.rb (hunk 2 above)
Comment (gabybrow) [reply app/controllers/api/v1/books_controller.rb]: Yes. It is returning a JSON with the same format as Apiary

File: app/controllers/api/v1/books_controller.rb (hunk 3 above)
Comment (gabybrow) [reply app/controllers/api/v1/books_controller.rb]: Same

**Review by Dinuuu — APPROVED** [review]

**Pull request merged**